nmtwizard/preprocess/tu.py

```diff
@@ -185,6 +188,34 @@ def output_side(self):
     def output_delimiter(self):
         return self._output_delimiter
 
+    def finalize(self):
+        _ = self.tok
```

Comment:
Looks like we also need to synchronize detok here, otherwise some tests do not pass because detokenization cannot be done without tokenizer.

Reply:
Just saw you fixed it :)